### PR TITLE
add python.twilio.security.twiml-injection

### PR DESCRIPTION
**Pull request by becojo**
Inspired by the the talk [_API: Alternate Pathway to Injection_ by Fennix at Northsec 2024](https://nsec.io/session/2024-api-alternate-pathway-to-injection.html), add a rule to detect a code sample in Twilio's documentation that could lead to injection of TwiML.

### CODE REVIEW

File: python/twilio/security/twiml-injection.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    message: >-
+      Using non-constant TwiML (Twilio Markup Language) argument when creating a
+      Twilio conversation could allow the injection of additional TwiML commands
+    metadata:
+      cwe:
+      - "CWE-91: XML Injection"
+      owasp:
+      - "A03:2021 - Injection"
+      category: security
+      technology: [--no-technology--]
```
Comment (inkz) [inline python/twilio/security/twiml-injection.yml]: Can you please add technology here? like "twilio", "twiml" etc

**Review by inkz — CHANGES_REQUESTED** [review]

**Review by inkz — APPROVED** [review]
Great job, thank you for your contribution. Would love to see the recording of the presentation :smile:

**Comment by becojo** [comment]
@inkz it's available in this live stream recording starting at ~47:00 https://www.youtube.com/live/YUJu8qkkz-o?feature=shared&t=2824

**Comment by inkz** [comment]
@becojo awesome, thank you very much :+1:

**Pull request merged**